pkg/apis/kubeone/v1beta2/types.go

```diff
@@ -882,6 +882,9 @@ type Addon struct {
 	// Params to the addon, to render the addon using text/template, this will override globalParams
 	Params map[string]string `json:"params,omitempty"`
 
+	// Disable templatization for the addon
+	DisableTemplate bool `json:"disableTemplate,omitempty"`
```

Comment:
I'm wondering if we should call this with `-ing`, like `DisableTemplating`. I don't have strong opinions, but that might be a better name.

Reply:
Good point, updated.